astropy/io/fits/diff.py

```diff
@@ -1209,16 +1280,16 @@ def _report(self):
                                                       self.diff_ratio))
 
 
-def diff_values(a, b, tolerance=0.0):
+def diff_values(a, b, rtol=0.0, atol=0.0):
     """
     Diff two scalar values.  If both values are floats they are compared to
-    within the given relative tolerance.
+    within the given absolute and relative tolerance.
     """
 
     if isinstance(a, float) and isinstance(b, float):
         if np.isnan(a) and np.isnan(b):
             return False
-        return not np.allclose(a, b, tolerance, 0.0)
+        return not np.allclose(a, b, rtol, atol)
```

Comment:
Could you pass them in by name here as well? `rtol=rtol, atol=atol`? That's what is done oion the other places as well.